spec/services/data_migrations/backfill_current_course_option_id_spec.rb

```diff
@@ -0,0 +1,21 @@
+require 'rails_helper'
```

Comment:
I guess now this has been merged we can remove from this commit with the data migration file?